examples/hello.jsonnet

```diff
@@ -1,4 +1,4 @@
 {
-    w:: "World",
-    "Hello": $.w
+  w:: 'World',
+  Hello: $.w,
```

Comment:
These examples have been formatted accidentally, but this makes me think: what's the role of example files? 
1. Are they examples of the output of formatters? (but some files are formatted and some are not)
2. Are they examples of *input* files to the formatters?

In either case I think we could turn them into diff tests, i.e. add pairs (unformatted file, expected output) for each formatter and test whether running bazel-super-formatter with all flags turned on transforms (unformatted file) into (expected output). Of course this will have a side effect of depending on specific formatting behavior exhibited by a given formatter, but in general what we want to test is whether the formatter is run at all by our meta-formatter